src/templates.c

```diff
@@ -283,6 +285,42 @@ static void create_file_template_menu(void)
 	g_object_ref(new_with_template_toolbar_menu);
 	geany_menu_button_action_set_menu(GEANY_MENU_BUTTON_ACTION(toolbar_get_action_by_name("New")),
 		new_with_template_toolbar_menu);
+
+	// create config files menu
+	item = gtk_menu_item_new_with_mnemonic(_("Templates"));
+	gtk_widget_show(item);
+	gtk_container_add(GTK_CONTAINER(ui_widgets.config_files_menu), item);
+	menu = gtk_menu_new();
+	gtk_menu_item_set_submenu(GTK_MENU_ITEM(item), menu);
+	list = utils_get_config_files(GEANY_TEMPLATES_SUBDIR);
+	foreach_slist(node, list)
+	{
+		gchar *fname = node->data;
+
+		SETPTR(fname, g_build_filename(app->configdir, GEANY_TEMPLATES_SUBDIR, fname, NULL));
+		if (!g_file_test(fname, G_FILE_TEST_IS_DIR))
+			ui_add_config_file_menu_item(fname, NULL, GTK_CONTAINER(menu));
+		g_free(fname);
+	}
+	g_slist_free(list);
+
+	item = gtk_menu_item_new_with_mnemonic(_("Files"));
```

Comment:
Shouldn't the Files submenu be also at the start of the list, similarly to what's done below in compare_menu_item_labels()?